app/services/CalculationResultService.scala

```diff
@@ -31,19 +32,32 @@ import java.time.LocalDate
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class CalculationResultService @Inject() (connector: CalculationResultConnector, auditService: AuditService)(implicit
+class CalculationResultService @Inject() (
+  calculationResultConnector: CalculationResultConnector,
+  backendConnector: BackendConnector,
+  auditService: AuditService
+)(implicit
   ec: ExecutionContext
 ) extends Logging {
 
   def sendRequest(userAnswers: UserAnswers)(implicit hc: HeaderCarrier): Future[CalculationResponse] =
     for {
       calculationUserAnswers <- Future.successful(buildCalculationUserAnswers(userAnswers))
-      calculationResponse    <- connector.sendRequest(calculationUserAnswers)
+      calculationResponse    <- calculationResultConnector.sendRequest(calculationUserAnswers)
       _                      <- auditService.auditCalculationSubmissionRequest(
                                   CalculationSubmissionAuditEvent(calculationUserAnswers, calculationResponse)
                                 )
     } yield calculationResponse
 
+  def submitUserAnswersAndCalculation(answers: UserAnswers): Future[SubmissionResponse] = {
+    val calculationUserAnswers: CalculationUserAnswers = buildCalculationUserAnswers(answers)
```

Comment:
Would this be more appropriately names as something like 'CalculationRequest' rather than calculation user answers as at this points its been transformed as is no longer UA. I realise this also applies to Sandips method below 'buildCalculationUserAnswers'

Reply:
Yes, agreed, have renamed to "CalculationInputs". Didn't want to use "CalculationRequest" as this is an overloaded name as used in the calculate backend for the request made to PAAC, and would cause confusion.

Reply:
Have changed backend in alignment also.